### PR TITLE
Update VS redist package version

### PR DESCRIPTION
**Pull request by RussKie**
Relates to https://github.com/dotnet/windowsdesktop/pull/3114

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7664)

### CODE REVIEW

File: Directory.Build.targets [hunk 1]
```diff
@@ -65,14 +65,14 @@
...
+      Condition=" '$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)'!='' And $(TargetFramework.StartsWith('net7.')) ">
+    <TargetingPackVersion>$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)</TargetingPackVersion>
     </FrameworkReference>
   </ItemGroup>
 
   <PropertyGroup>
-    <!-- Set RuntimeFrameworkVersion to VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion from Core Setup to prevent large SDK cycle -->
-    <RuntimeFrameworkVersion Condition="'$(VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion)'!='' And $(TargetFramework.StartsWith('net7.')) ">$(VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion)</RuntimeFrameworkVersion>
+    <!-- Set RuntimeFrameworkVersion to VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion from Core Setup to prevent large SDK cycle -->
+    <RuntimeFrameworkVersion Condition="'$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)'!='' And $(TargetFramework.StartsWith('net7.')) ">$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)</RuntimeFrameworkVersion>
```
Comment (kasperk81) [inline Directory.Build.targets:75]: why there is '80' (.NET 8.0 version) in the name? is it a known convention of redist packages?

Comment (dreddy-work) [reply Directory.Build.targets:75]: @RussKie , if we are not maintaining/Using older versions, we may want to remove version suffix from name.

Comment (dreddy-work) [reply Directory.Build.targets:75]: if it is necessary, i think we should give it a consistant name across repos . ex: https://github.com/dotnet/installer/pull/14402/files

Comment (kasperk81) [reply Directory.Build.targets:75]: i think installer is following winforms, not the other way around. version suffix in name is strange, when there is a specific version field. even running multiple  versions side by side should not require changing naming convention of packages.

Comment (RussKie) [reply Directory.Build.targets:75]: > why there is '80' (.NET 8.0 version) in the name? is it a known convention of redist packages?

AFAIK this is an established naming convention for package references - i.e., _package name without "." + "PackageVersion" suffix_. This was introduced in https://github.com/dotnet/winforms/pull/4217.
Given the way this variable is used, having the version in the name makes it very obvious what version of the redistributable package we're referencing.




> if it is necessary, i think we should give it a consistant name across repos . ex: [dotnet/installer#14402 (files)](https://github.com/dotnet/installer/pull/14402/files)

I'm not very clear what exactly you're referring too, but the installer repo needs to update some of its definitions.

File: docs/testing-in-vs.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ There are two ways to get around this problem:
     * If you’re interested in seeing what this version is, look at `<yourRepoRoot>\.dotnet\sdk\<yourSdkVersion>\dotnet.runtimeconfig.json`
 * The winforms (runtime) repo DOES specify an explicit NetCore.App version, and we use the one published by the core-setup repo.
     * If you look at [Directory.Build.Targets](https://github.com/dotnet/winforms/blob/ac0426561b158522eb8564de2bedd28f28148f8d/Directory.Build.targets#L35), you can see where this is set 
-* `$(VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion)` is set in [Versions.props](https://github.com/dotnet/winforms/blob/ac0426561b158522eb8564de2bedd28f28148f8d/eng/Versions.props#L19), and gets automatically updated when we ingest dependencies from core-setup. (These PR's are automatically merged in if the build passes)
+* `$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)` is set in [Versions.props](https://github.com/dotnet/winforms/blob/ac0426561b158522eb8564de2bedd28f28148f8d/eng/Versions.props#L19), and gets automatically updated when we ingest dependencies from core-setup. (These PR's are automatically merged in if the build passes)
```
Comment (dreddy-work) [inline docs/testing-in-vs.md:47]: > VS.Redist.Common.WindowsDesktop.SharedFramework.x64.8.0

I am not sure why we are forced to update this manually. We never did this before and expected to be updated from the dependency flow . Do you know what exactly broke this time?

**Review by dreddy-work — APPROVED** [review]

File: docs/testing-in-vs.md (hunk 2 above)
Comment (RussKie) [reply docs/testing-in-vs.md:47]: This is a markdown file. And yes, we did update this manually before - https://github.com/dotnet/winforms/pull/5459.

**Pull request merged**